### PR TITLE
Update HACKING to include openssl dependency and correct command for …

### PR DESCRIPTION
**Pull request by AlexTheAussieNerd**
…forked repositories

I've re-run the install on a different machine and noted some more changes were needed. 

No download would happen on the existing git clone command for forked repositories.

Building as per instructions failed on cmake? saying a dependeny wasn't met for openssl.

### CODE REVIEW

**Comment by petitlapin** [comment]
Thanks for testing and sorry for delay in answering,

libssl should not be required at compile time (it is an optional dependency at runtime if you need to download the assets) so cmake should not complain about it. Do you have the error?

**Comment by AlexTheAussieNerd** [comment]
I've just re-run and can confirm the error was reported after running "cmake .." (on a raspberry pi 400, latest 64bit os)

This isnt exactly the error message as I'm typing from another screen, but the numbers will be identical.

The source was downloaded from a git clone https:// official mirror repository


The following REQUIRED packages have been found:
 * Qt <lots of packages>
 
- The following OPTIONAL packages have not been found 
- ECM (required version > 1.4.0)
-
- The following REQUIRED packaged have not been found:
 Open SSL

Cmake Error at /usr/share/cmake-3.25/Modules/FeatureSummary.cmake:464 (message: 
  feature_summary() Error: REQUIRED packages(s) are missiong, aborting CMake run.
Call Stack (most recent call first):
 CMakeLists.txt:143 (feature_summary)

- Configuring incomplete, errors occurred!
- See also "..GCompris-qt/build/CMakeFiles/CMakeOutput.log"  
- (CMakeOutput.log didnt show and error about it thought).

**Comment by petitlapin** [comment]
You're right (https://invent.kde.org/education/gcompris/-/blob/master/CMakeLists.txt#L132-135), I'll add the dependency in the README when I have 5 minutes. Thank you!

File: HACKING [hunk 1]
```diff
@@ -20,7 +20,7 @@ Checkout the source code by cloning it from GitHub
 If you want to have a personal fork, in order to commit occasional patches,
 fork it on GitHub then clone your fork
 
-    git clone git@github.com:<username>/GCompris-qt.git
```
Comment (petitlapin) [inline HACKING:23]: this works if you have a ssh key (and this allows you to push without inputting your account/password everytime)

**Comment by petitlapin** [comment]
I've just pushed it in https://invent.kde.org/education/gcompris/-/commit/2b7989ab54c461efb49a55c4fc9de7286c99a07b.

Thank you!

**Pull request closed without merging**